service.py

```diff
@@ -302,33 +303,42 @@ def createListItemFromFlatPlaylistItem(video):
 
     return listItem
 
-# get the index of the first video to be played in the submitted playlist url
 def playlistIndex(url, playlist):
-    if sys.version_info[0] >= 3:
-        from urllib.parse import urlparse, parse_qs
-    else:
-        from urlparse import urlparse, parse_qs 
```

Comment:
remove conditional, instead use python2 import because we are on the leia branch